src/cromshell/utilities/VariableSetup.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+
+def get_submission_file(config_dir):
+    submission_file = "{}/all.workflow.database.tsv".format(config_dir)
+    if not Path.exists(Path(submission_file)):
+        Path.touch(Path(submission_file))
+        submission_header = 'DATE{0}CROMWELL_SERVER{0}RUN_ID{0}WDL_NAME{0}STATUS{0}ALIAS'.format('\t')
+        with Path(submission_file).open() as f:
+            f.write(submission_header)
+    return submission_file
+
+
+def get_config_dir():
+    config_path = str(Path.home()) + "/.cromshell"
+    Path.mkdir(Path(config_path), exist_ok=True)
+    return config_path
+
+
+def get_cromwell_server(config_dir):
```

Comment:
I believe these get methods should be living inside the `Cromwell` class.

Reply:
done